src/rpc/rawtransaction.cpp

```diff
@@ -939,7 +939,7 @@ static UniValue signrawtransactionwithkey(const JSONRPCRequest& request)
                 "this transaction depends on but may not yet be in the block chain.\n",
                 {
                     {"hexstring", RPCArg::Type::STR, false},
-                    {"privkyes", RPCArg::Type::ARR,
+                    {"privkeys", RPCArg::Type::ARR,
```

Comment:
Reported by @kewde http://www.erisian.com.au/bitcoin-core-dev/log-2018-11-26.html#l-139.

Reply:
Good catch. Currently the argument names of arrays are not used in any way to produce the string by RPCHelpMan, only after #14796.